frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -2445,6 +2445,12 @@ function ReaderFooter:onNetworkConnected()
 end
 ReaderFooter.onNetworkDisconnected = ReaderFooter.onNetworkConnected
 
+function ReaderFooter:onPageTurningInverted()
```

Comment:
```suggestion
function ReaderFooter: onSwapPageTurnButtons()
```

Reply:
Why? Is that coding style used elsewhere?

Reply:
Never mind, I was fixating on the stray space after the `:`.